### ISSUE TITLE
English translation missing for "Eine Verlängerung ist noch nicht möglich."

### ISSUE THREAD

**Issue by johnjohndoe** [raphaelm/opacclient#309]
I am running my phone on English. I tried to extend a book and noticed that the popup message `Eine Verlängerung ist noch nicht möglich.` appears in German as shown in the screenshot.

![Web Opac popup](https://cloud.githubusercontent.com/assets/144518/7770612/9ff2a30c-0090-11e5-9056-c9c945be05cb.png)

Since I could not find the string in the repository - I suspect it is a response of the backend. You might wanna intercept the response to translate it on the client.

**Comment by raphaelm** [comment]
Hi Tobias,

we are definitely not going to maintain a catalgoue of all strings all >800 libraries might ever send in multiple languages. Sorry ;)

We will, however, look into possibilities to tell the server to response in English when the library's catalogue does have an English version, as the HTW Berlin's does. We do not do this currently, as it would break some parts of the parser.

**Comment by johnjohndoe** [comment]
Maybe there is header information which are helpful. Whatsoever, it is not such as big thing that it distracts me. I just wanted to let you know about it.

**Comment by raphaelm** [comment]
Yes, thank you :) It might be distracting to people who do not speek German, but use a German library, like foreign students, so I do count it as an issue to work on but not an urgent one.

Unfortunately, there is no header information or anything similar. There is not even anything structured in the response at all. To get a grasp of how the app works in this case: We take the text response of the library and present it to the user and decide whether it was a success or not based on the CSS class used for the message…

**Comment by johnjohndoe** [comment]
*/phuu/* nothing to add ...
